src/vs/workbench/contrib/accessibilitySignals/browser/accessibilitySignalLineFeatureContribution.ts

```diff
@@ -177,11 +181,11 @@ interface LineFeature {
 
 interface LineFeatureSource {
 	isPresent(position: Position, reader: IReader): boolean;
+	isPresentOnLine?(position: Position, reader: IReader): boolean;
```

Comment:
optional bc most instances of `isPresent` do not deal with `position.column`, only `position.lineNumber`

Reply:
> optional bc most instances of isPresent do not deal with position.column, only position.lineNumber

If they don't deal with the column, maybe they should only implement `isPresentOnLine` then?
Also, `isPresentOnLine` should only get the line number.

Maybe `LineFeatureSource` should be a class and `isPresent` should have a default implementation:
```ts
abstract class LineFeatureSource {
	isPresent(position: Position, reader: IReader): boolean { return this.isPresentOnLine(position.lineNumber, reader); }
	abstract isPresentOnLine(lineNumber: number, reader: IReader): boolean;
}
```
The default implementation of `isPresent` could also be to return `false`.

Reply:
default implementation of `isPresent` false so that for breakpoint etc would only look at the line